analyzer/pkg/rpm/rpm.go

```diff
@@ -30,6 +30,8 @@ var requiredFiles = []string{
 }
 var errUnexpectedNameFormat = xerrors.New("unexpected name format")
 
+var defaultOsVendors = []string{"Amazon Linux", "Amazon.com", "CentOS", "Fedora Project", "Oracle America", "Red Hat, Inc.", "AlmaLinux", "CloudLinux"}
```

Comment:
"default" may be a bit confusing. `osVendors` looks enough to me.

```suggestion
var osVendors = []string{"Amazon Linux", "Amazon.com", "CentOS", "Fedora Project", "Oracle America", "Red Hat, Inc.", "AlmaLinux", "CloudLinux"}
```

Reply:
Could you add which OS the vendor refers to as a comment so that we can easily identify which OS is missing? Looks like Photon OS and SUSE are missing.

```suggestion
var defaultOsVendors = []string{
    "Amazon Linux", // Amazon Linux 1
    "Amazon.com", // Amazon Linux 2
    "CentOS", // CentOS
    "Fedora Project", // Fedora
    "Oracle America", // Oracle Linux
    "Red Hat, Inc.", // Red Hat
    "AlmaLinux", // AlmaLinux
    "CloudLinux" // ??
}
```